Formula/chamber.rb

```diff
@@ -0,0 +1,31 @@
+class Chamber < Formula
+  desc "CLI for managing secrets through AWS SSM Parameter Store"
+  homepage "https://github.com/segmentio/chamber"
+  url "https://github.com/segmentio/chamber.git",
+    :tag => "v1.9.0",
+    :revision => "acc4072df92f2763217ccfa5373e14f342e178ba"
+  sha256 "098568ddda04956b8dc270aa8e6b4eb83d8cfe92add8ea0172715eab7db050cb"
+  head "https://github.com/segmentio/chamber.git"
+
+  depends_on "go" => :build
+  depends_on "govendor" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = "amd64"
+    ENV["CGO_ENABLED"] = "0"
+
+    path = buildpath/"src/github.com/segmentio/chamber"
+    path.install Dir["{*,.git}"]
+
+    cd buildpath/"src/github.com/segmentio/chamber" do
+      system "govendor", "sync"
+      system "go", "build", "-o", bin/"chamber"
+    end
+  end
+
+  test do
+    system bin/"chamber", "-h"
```

Comment:
New formulae in homebrew-core are required to have a substantive test, not just `help` or `version`. Could you come up with one here?

Reply:
I'd be more than happy to, I'm just not sure how since all functionality requires AWS keys. Is there a policy for formulae like this? Would it be sufficient to assert the error returned?

Reply:
cc @ilovezfs for an opinion here.

Reply:
@ilovezfs any suggestions on how to best test this formula? Thanks!